tests/staticfiles_tests/test_storage.py

```diff
@@ -487,6 +487,18 @@ def test_template_tag_simple_content(self):
             self.assertIn(b"other.deploy12345.css", content)
 
 
+@override_settings(STATICFILES_STORAGE='staticfiles_tests.storage.DontHashReferenceStorage')
+class TestCollectionNoneHashStorage(CollectionTestCase):
+    run_collectstatic_in_setUp = False
+
+    def test_collectstatistic_runs_without_error(self):
+        err = StringIO()
+        try:
+            call_command('collectstatic', interactive=False, verbosity=0, stderr=err)
+        except UnboundLocalError as e:
+            self.fail(f"Collectstatic command raised {e.__class__.__name__} unexpectedly!")
```

Comment:
I changed this to an assertion for the only file that is affected by the second round of post-processing i.e. `cached/relative.css`.